apps/project/tests/test_apis.py

```diff
@@ -938,76 +971,18 @@ def test_auto_accept(self):
         self.assertEqual(request.status, 'accepted')
         self.assertEqual(request.responded_by, self.user)
 
-    def _test_status_filter(self, and_conditions):
-        status = self.create(ProjectStatus, and_conditions=and_conditions)
-        self.create(
-            ProjectStatusCondition,
-            project_status=status,
-            condition_type=ProjectStatusCondition.NO_LEADS_CREATED,
-            days=5,
-        )
-        self.create(
-            ProjectStatusCondition,
-            project_status=status,
-            condition_type=ProjectStatusCondition.NO_ENTRIES_CREATED,
-            days=5,
-        )
-
-        old_date = timezone.now() - timedelta(days=8)
-
-        # One with old lead
-        project1 = self.create(Project, role=self.admin_role)
-        lead = self.create(Lead, project=project1)
-        Lead.objects.filter(pk=lead.pk).update(created_at=old_date)
-        Lead.objects.get(pk=lead.pk).save()
-
-        # One with latest lead
-        project2 = self.create(Project, role=self.admin_role)
-        self.create(Lead, project=project2)
+    def test_status_filter(self):
```

Comment:
Let's add a test for updating status aswell.

Reply:
Done